pages/partners.vue

```diff
@@ -41,8 +40,23 @@
 
 <script setup lang="ts">
 import SignupBox from '@/components/forms/SignupBox.vue'
+import { asLink } from '@prismicio/helpers'
+
+const { client } = usePrismic()
+const { data: partners } = await useAsyncData('partner', () =>
+  client.getSingle('partnerspage', {
+    fetchLinks: ['accordionitem.title', 'accordionitem.slices']
+  })
+)
+usePrismicSEO(partners?.value?.data)
 
-const partners = [
+const partnersData = partners?.value?.data
+const partnersList = partners?.value?.data?.partners
+
+</script>
+
+/*
```

Comment:
once you've added all the data to prismic, would be great to then remove the commented out list of partners